pom.xml

```diff
@@ -1565,6 +1565,10 @@
                         <groupId>org.slf4j</groupId>
                         <artifactId>slf4j-log4j12</artifactId>
                     </exclusion>
+                    <exclusion>
```

Comment:
Could you please elaborate on why this exclusion is needed?

Reply:
I got this error message after building the project:
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/Users/mbalazs/projects/test/bin/../dep/slf4j-reload4j-1.7.35.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/mbalazs/projects/test/bin/../dep/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
log4j:WARN No appenders could be found for logger (org.apache.hadoop.metrics2.lib.MutableMetricsFactory).
log4j:WARN Please initialize the log4j system properly.
```
I checked with `mvn dependency:tree` and the hadoop-auth and hadoop-common dependency pulled in the new `slf4j`.

Reply:
@zeroflag - Does this exclusion seem OK from logging perspective?

Reply:
The `slf4j-reload4j` is the same as `slf4j-log4j12` but for reload4j (which is a up to date version of log4j one). Given that hadoop or other components might use slf4j for logging instead log4j (what knox uses) we need some kind of bridging library between those two. But this is for log4j1. So slf4j knows about how to log using Log4j/reload4j. But knox uses log4j2. So I don't think this is needed. I think the exclusion is fine.